core/tests/test_utils.py

```diff
@@ -340,31 +339,70 @@ def set_config_property(self, config_obj, new_config_value):
 
         self._restore_stashed_user_env()
 
+    def set_user_role(self, user_name, user_role):
+        """Sets the given role for this user.
+
+        Args:
+            user_name: str. Username of the given user.
+            user_role: str. Role of the given user.
+        """
+        self._stash_current_user_env()
+
+        self.login('tmpsuperadmin@example.com', is_super_admin=True)
+        response = self.testapp.get('/admin')
+        csrf_token = self.get_csrf_token_from_response(response)
+        self.post_json('/adminrolehandler', {
+            'username': user_name,
+            'role': user_role
+        }, csrf_token)
+        self.logout()
+
+        self._restore_stashed_user_env()
+
     def set_admins(self, admin_usernames):
-        """Set the ADMIN_USERNAMES property."""
-        self.set_config_property(
-            config_domain.ADMIN_USERNAMES, admin_usernames)
+        """Sets role of given users as ADMIN.
+
+        Args:
+            admin_usernames: list(str). List of usernames.
+        """
+        for name in admin_usernames:
+            self.set_user_role(name, feconf.ROLE_ID_ADMIN)
 
     def set_moderators(self, moderator_usernames):
-        """Set the MODERATOR_USERNAMES property."""
-        self.set_config_property(
-            config_domain.MODERATOR_USERNAMES, moderator_usernames)
+        """Sets role of given users as MODERATOR.
+
+        Args:
+            moderator_usernames: list(str). List of usernames.
+        """
+        for name in moderator_usernames:
+            self.set_user_role(name, feconf.ROLE_ID_MODERATOR)
 
     def set_banned_users(self, banned_usernames):
-        """Set the BANNED_USERNAMES property."""
-        self.set_config_property(
-            config_domain.BANNED_USERNAMES, banned_usernames)
+        """Sets role of given users as BANNED_USER.
+
+        Args:
+            banned_usernames: list(str). List of usernames.
+        """
+        for name in banned_usernames:
+            self.set_user_role(name, feconf.ROLE_ID_BANNED_USER)
 
     def set_collection_editors(self, collection_editor_usernames):
-        """Set the WHITELISTED_COLLECTION_EDITOR_USERNAMES property."""
-        self.set_config_property(
-            config_domain.WHITELISTED_COLLECTION_EDITOR_USERNAMES,
-            collection_editor_usernames)
+        """Sets role of given users as COLLECTION_EDITOR.
+
+        Args:
+            collection_editor_usernames: list(str). List of usernames.
+        """
+        for name in collection_editor_usernames:
+            self.set_user_role(name, feconf.ROLE_ID_COLLECTION_EDITOR)
 
     def set_email_senders(self, email_sender_usernames):
-        """Set the WHITELISTED_EMAIL_SENDERS property."""
-        self.set_config_property(
-            config_domain.WHITELISTED_EMAIL_SENDERS, email_sender_usernames)
+        """Sets role of given users as ADMIN.
```

Comment:
Eh, let's just drop this method and use set_admins() in place of set_email_senders() where needed.